docs/blitz/design/advanced_actions/advanced.rst

```diff
@@ -555,11 +559,100 @@ Examples are provided below for these conditional statements.
                      arguments:
                        interface: GigabitEthernet10
                 - run_condition:
-                       if: "%VARIABLES{gims_output} == 1500 and %VARIABLES{gims_output} == 2500"       
+                       if: "%VARIABLES{gims_output} == 1500 and %VARIABLES{gims_output} == 2500"
                        actions:
                          - parse:
                              command: show version
                              device: uut
                          - sleep:
                              sleep_time: 1
 
+
+**Example-6: Running actions with if else condition**
+
+.. code-block:: YAML
+
+    test:
+        source:
+            pkg: genie.libs.sdk
+            class: triggers.blitz.blitz.Blitz
+        devices: ['uut']
+        test_sections:
+            - apply_config:
+                - api:                                              # api output is equal to 1500
+                     device: uut
+                     function: get_interface_mtu_size
+                     save:
+                       - variable_name: gims_output                 # the 1500 is stored in gims_output

```

Comment:
`gims_output`/`gims_output_1` -> `mtu1`/`mtu2` is better

Reply:
updated